_posts/2024-08-01-contributing-to-che-docs.adoc

```diff
@@ -0,0 +1,51 @@
+---
+title: Contributing to Eclipse Che documentation
```

Comment:
```suggestion
title: Contributing to Eclipse Che documentation with one click
```

Reply:
not ideal, so feel free to come up with smth. better but as a reader, we should have smth. in the title that would grab the attention (the current one is very neutral for the blog imo)